src/entities/expenses/ui/ExpensesList/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { type FC } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { expensesApi } from '@entities/expenses/api';
+import { useStore } from '@shared/lib/useStore';
+import { Expense } from '../Expense';
+
+export const ExpensesList: FC = () => {
+  const { expensesStore, userStore } = useStore();
+
+  const { isLoading, isError, data, error } = useQuery({
```

Comment:
Explained about business logic above
Should separate it in `./models` and use `useExpensesQuery()`